core/migrations/0029_userprofile.py

```diff
@@ -48,6 +48,14 @@ class Migration(migrations.Migration):
                         verbose_name="Show locked",
                     ),
                 ),
+                (
+                    "new_progress",
```

Comment:
I'd rename this from `new_progress` to like `next_progress` or something. IIRC Evan suggests that this is actually older than the current system.